style.css

```diff
@@ -381,3 +405,31 @@ body {
 
 /* KONIEC ANIMOWANEGO MENU  */
 
+.joke-text {
+  width: 500px;
+  flex-shrink: inherit;
+  background-color: white;
+  padding: 8px;
+  border-radius: 8px 1px 8px 8px;
+  margin: 6px 0px;
+}
+
+.question-text {
+  width: 500px;
+  flex-shrink: inherit;
+  background-color: #654F6F;
+  color: white;
+  padding: 8px;
+  border-radius: 1px 8px 8px;
+  margin: 6px 0px;
+}
+
+.answer-text {
+  width: 500px;
+  flex-shrink: inherit;
+  background-color: white;
+  padding: 8px;
+  border-radius: 8px 1px 8px 8px;
```

Comment:
All of the class you wrote so 

joke, question, answer are almost the same. In order to prevent the css file to extend even longer (430 lines is quite long) i would suggest to make the 1 class that takes all the similarities and merge them in 1 class. You can still of course leave the class answer question and joke in order to easier mange them from js lvl, but i would also suggest add something like text-box that will focus on their css. Only the differences should be put in the joke, quesiton, answer classes.